src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -529,7 +532,7 @@ private Locomotive performSwipe(SwipeElementDirection direction, WebElement elem
     private Locomotive performCornerSwipe(ScreenCorner corner, SwipeElementDirection direction, float percentage, int duration) {
         Dimension screen = getAppiumDriver().manage().window().getSize();
 
-         final int SCREEN_MARGIN = 10;
+         final int SCREEN_MARGIN = 130;
```

Comment:
Changing the margin seems unrelated to the original intent of this PR. A better solution than increasing it for everyone would also be to overload `performCornerSwipe()` with an additional parameter to overwrite the default screen margin.
Could we handle this in a separate PR?

Reply:
yeah, this wasn't meant to be in this pr